programs/igt/meta.yaml

```diff
@@ -0,0 +1,345 @@
+description: >
```

Comment:
Pls add "short_description: >" for all applicable tests.

Reply:
ok